src/components/Program/Program.module.css

```diff
@@ -0,0 +1,147 @@
+.program {
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
+  padding-top: 32px;
+  padding-bottom: 126px;
+  background-image: url(../../assets/bgImages/program.png);
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+.container {
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+  align-content: space-around;
+  flex-wrap: wrap;
+  padding: 0 20px;
```

Comment:
ten padding przeniósłbym do .program i dla przełamań kolejno pozmieniał jak w projekcie bo jak sprawdzałem to na pc jest wysrodkowany program a jak klikniesz na ptaszka do się poszerza co dla mnie nieciekawie wygląda.
Głównie ten problem wychodzi na PC na tablecie i mobilce jest ok. Przejrzyj to proszę :D